app/controllers/UnsecuredApplication.java

```diff
@@ -0,0 +1,58 @@
+package controllers;
+
+import play.Configuration;
+import play.Play;
+import play.db.jpa.Transactional;
+import play.mvc.Result;
+import play.mvc.Controller;
+
+
+public class UnsecuredApplication extends Controller {
+
+   	private static String INFO = null;
+
+    public static String info() {
+    	init();
+        return INFO;
+    }
+
+   	private static void init(){
+   		if (INFO == null){
+	   	    final Configuration cfg = Play.application().configuration();
+			String version = "Version: " + cfg.getString("app.version");
+	   	    String dbName = "Database: " + cfg.getString("db.default.url");
+	   	   	INFO = "Copyright 2014-2016 - University of Pittsburgh, " 
+	   	    + version + ", " + dbName;
+   		}
+   	}
+   	
+   	
+   	public Result about() {
+		return ok(views.html.about.render("about", info()));
+	}
```

Comment:
Ah, okay. I left those methods in because at one point there was interest in making LS searchable without an account.

I think Jeremy was saying he envisioned login more so for customization purposes rather than access to the site.